src/GravitationalClustering.js

```diff
@@ -65,6 +48,7 @@ export default class GravitationalClustering {
         this.eps = Number.MAX_VALUE;
```

Comment:
that's confusion. "eps" usually means epsilon and epsilon is a very small number, not the maximum value!